iphone/Classes/TiContactsPerson.m

```diff
@@ -618,7 +611,7 @@ -(id)valueForUndefinedKey:(NSString *)key
 #if IS_XCODE_7
 	if ([TiUtils isIOS9OrGreater]) {
 		//birthday property managed seperately
-		if ([key isEqualToString:@"birthday"]) {
+		if ([key isEqualToString:@"birthday"] && [person isKeyAvailable:CNContactBirthdayKey]) {
```

Comment:
I just tested by selecting the person's birthday in showContacts. It doesn't return the date because isKeyAvailable is returning no. It looks like if you select birthday, apple doesn't return you the value. If this is the case, and it's native behavior, we can just document about it under `selectedProperty` in apidoc